src/main/java/eu/tornplayground/tornapi/limiter/DefaultRequestLimiter.java

```diff
@@ -0,0 +1,11 @@
+package eu.tornplayground.tornapi.limiter;
+
+/**
+ * Default implementation of {@link RequestLimiter} with a limit of 100 requests per minute per API key.
+ * That's the default limit for the Torn API.
+ */
+public class DefaultRequestLimiter extends RequestLimiter {
```

Comment:
Not really a fan of implying this as the default, as there is no need to space out requests evenly by default.